nums/numpy/api/__init__.py

```diff
@@ -0,0 +1,104 @@
+# coding=utf-8
+# Copyright (C) 2020 NumS Development Team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# pylint: disable = redefined-builtin, too-many-lines, W1401, W0401, W0614
```

Comment:
Had to add a few more pylint ignores to pass the pylint check. Let me know if there is a better way to do this or if some of these checks aren't needed.

Added flags:
* **W1401 (anomalous-backslash-in-string)**
  * Anomalous backslash in string: ‘%s’. String constant might be missing an r prefix. Used when a backslash is in a literal string but not as an escape.
* **W0401 (unused-wildcard-import)**
  * Unused import %s from wildcard import Used when an imported module or variable is not used from a ‘from X import *’ style import.
* **W0614 (wildcard-import)**
  * Wildcard import %s Used when from module import * is detected.

Source: https://docs.pylint.org/en/1.6.0/features.html